database/seeders/DatabaseSeeder.php

```diff
@@ -13,11 +14,49 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // User::factory(10)->create();
-
-        User::factory()->create([
-            'name' => 'Test User',
-            'email' => 'test@example.com',
-        ]);
+        $users = [
+            [
+                'name' => 'Pasha',
+                'email' => 'www@PashkaPASS555777',
+                //         'password' => '555777',
+                'email_verified_at' => NULL,
+                // hash ниже
+                'password' => '$2y$12$px9y5AgGxGjSOMKZ9vATN.6dDSYJ0UV6gsDns8YHbdEsLmcKSHd0W',
+                'remember_token' => NULL,
+            ],
+            [
+                'name' => 'Dany',
+                'email' => 'www@DanykaPASS123123',
+                //         'password' => '123123',
+                'email_verified_at' => NULL,
+                // hash ниже
+                'password' => '$2y$12$JYokrB8a7Fe53.VUL3V3EesskuSb5luO8F/pz3OB8SjHxdxRKS9.G',
+                'remember_token' => NULL,
+            ],
+            [
+                'name' => 'Vova',
+                'email' => 'www@VovanPass365365',
+                //         'password' =>'365365',
+                'email_verified_at' => NULL,
+                // hash ниже
+                'password' => '$2y$12$AQqA5EpxrCzML8XnEwbb3O7QkSu5hYHJLKMEuwYOkcOKDlxSnq5D.',
+                'remember_token' => NULL,
+            ],
+        ];
+        for ($i=0; $i < count($users); $i++) {
+            User::factory(1, [
```

Comment:
please, use User::factory()->count(...) to create as many users as you want. 

Remove the for loop. It's the most complex solution for arrays looping
https://www.php.net/manual/en/control-structures.for.php